Programowanie Obiektowe/ProgramowanieObiektoweReszta/_1_12RoznicaPomiedzyKlasaAFigura/Figura.cs

```diff
@@ -0,0 +1,16 @@
+﻿namespace _1_12RoznicaPomiedzyKlasaAStruktura
+{
+    public interface Figura
+    {
+        abstract public void WyswietlWzor();
```

Comment:
W interfejscie nie musisz pisać abstract ani modyfikatora dostępnu.
```suggestion
        void WyswietlWzor();
```

Reply:
[cb7a4dc](https://github.com/broniuu/Praktyki2022/pull/2/commits/cb7a4dc8fb7e1041a4c4485dde74d2bfb24dcfb6)